runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -1573,6 +1604,30 @@ gcParseCommandLineAndInitializeWithValues(J9JavaVM *vm, IDATA *memoryParameters)
 		memoryParameters[opt_Xmox] = memoryParameters[opt_Xmo];
 	}
 
+	result = option_set_to_opt_double(vm, OPT_XXMAXRAMPERCENT, &index, EXACT_MEMORY_MATCH, &extensions->maxRAMPercent);
+	if (OPTION_OK != result) {
+		goto _error;
+	}
+	memoryParameters[opt_maxRAMPercent] = index;
+	if (memoryParameters[opt_maxRAMPercent] != -1) {
+		if ((extensions->maxRAMPercent < 0.0) || (extensions->maxRAMPercent > 100.0)) {
+			j9nls_printf(PORTLIB, J9NLS_ERROR, J9NLS_GC_OPTIONS_DOUBLE_OUT_OF_RANGE, "-XX:MaxRAMPercentage", 0.0, 100.0);
```

Comment:
Can we reuse J9NLS_GC_OPTIONS_PERCENT_OUT_OF_RANGE ? Currently there is no need to have general "Double out-of-range" message.